app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p style="color: green"><%= notice %></p>
+
+<div class="recipe_containers">
+    <h2 class="recipe_title">
+      <%= @recipe.name %>
+    </h2>
+</div>
+<div class ="borders">
+<div class="recipe_info">
+      <p>Preparation time: <%= @recipe.preparation_time %> minutes<p>
+      <p>Cooking time: <%= @recipe.cooking_time %> minutes</p>
+      <p>Steps go here...: <%= @recipe.description %></p>
+</div>
+
+<div class = "recipe_contr">
+  <%= link_to "Edit this recipe", edit_recipe_path(@recipe), class: "btn-generate" %> 
+  <%= link_to "Destroy this recipe", @recipe, class: "btn-delete-recipe", method: :delete %>
+</div>
+
+<div class="recipe_contr">
+<%= link_to "Generate shopping list", general_shopping_list_path, class: "btn-generate", method: :get %>
+<%= link_to "Add ingredients", new_ingredient_path(@recipe), class: "btn-add_new_recipe", method: :get  %>
+</div>
+
+<% if @ingredients.empty? %>
+        <p class="no_item">No Ingredients Found! Please Add a new one.</p>
+    
+    <% else %>
+    <div class ="table_border">
+      <table class="paleBlueRows recipe_table">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @ingredients.each do |recipe_food| %>
+      <tr>
```

Comment:
- [ ] kindly try to match the wireframe given, I can not see a button to toggle the public recipe, also the button destroy and edit is not necessary as it is not in the wireframe. kindly adjust this view
in your design :point_down: 
![Screenshot from 2023-07-07 11-17-03](https://github.com/Ridwanullahi-code/RecipeApp/assets/66097371/b9e48058-df55-4b20-999b-502dcb19a1b5)
wireframe :point_down: 
![Screenshot from 2023-06-29 12-00-33](https://github.com/Ridwanullahi-code/RecipeApp/assets/66097371/f8d328b7-4535-4f91-9839-8428d1bd8c9a)

Reply:
Fixed 👌🏻